core/src/main/java/io/cucumber/core/plugin/UrlReporter.java

```diff
@@ -0,0 +1,39 @@
+package io.cucumber.core.plugin;
+
+import io.cucumber.core.exception.CucumberException;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.net.URL;
+
+class UrlReporter {
+    private static final String CYAN = "\u001B[36m";
+    private static final String RESET = "\u001B[0m";
+    private final Writer out;
+
+    public UrlReporter(Writer out) {
+        this.out = out;
+    }
+
+    public void report(URL url) {
+        String path = url.getPath();
+        int pathLength = path.length();
+        try {
+            out.append(CYAN).append("┌─────────────────────────────").append(times('─', pathLength)).append("┐")
```

Comment:
Plugins that use ansi colors should implement the `ColorAware` interface so as to avoid printing ansi symbols when this is unwanted.